homeassistant/components/bmp280/sensor.py

```diff
@@ -0,0 +1,161 @@
+"""Platform for Bosch BMP280 Environmental Sensor integration."""
+import logging
+
+from adafruit_bmp280 import Adafruit_BMP280_I2C
+from busio import I2C
+import voluptuous as vol
+
+from homeassistant.components.sensor import (
+    DEVICE_CLASS_PRESSURE,
+    DEVICE_CLASS_TEMPERATURE,
+    PLATFORM_SCHEMA,
+)
+from homeassistant.const import CONF_NAME, PRESSURE_HPA, TEMP_CELSIUS
+from homeassistant.exceptions import PlatformNotReady
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+
+_LOGGER = logging.getLogger(__name__)
+
```

Comment:
Is the default SCAN_INTERNAL of 15s ok here?
```
helpers/entity_component.py:DEFAULT_SCAN_INTERVAL = timedelta(seconds=15)
```

Reply:
Yes, however I'm adding this statement explicitly so it will stay 15s even if the default changes in `entity_component.py`